apps/web/components/DataLinkCard/DataLinkCard.treat.ts

```diff
@@ -0,0 +1,18 @@
+import { style, styleMap } from 'treat'
+import { theme } from '@island.is/island-ui/theme'
+
+export const card = style({
+  display: 'flex',
+  height: '100%',
+  width: '100%',
+  flexDirection: 'column',
+  cursor: 'pointer',
+  boxSizing: 'border-box',
+  minHeight: 146,
+  textDecoration: 'none',
+  position: 'relative',
```

Comment:
For maximum reusability we should use Box props as much as we can for styling, Box props are already generated to css classes and reused for every instance of Box
1/2
```suggestion
  boxSizing: 'border-box',
  minHeight: 146,
  textDecoration: 'none',
```